test/integration/patients/worklist/worklist.js

```diff
@@ -339,7 +339,7 @@ context('worklist page', function() {
       .wait('@routeFlows')
       .itsUrl()
       .its('search')
-      .should('contain', `filter[updated_at]=${ dayjs(testTs()).startOf('day').subtract(1, 'month').format() }`)
+      .should('contain', `filter[updated_at]=${ dayjs().startOf('day').subtract(30, 'days').format() }`)
```

Comment:
Tried to get too cute when I added this. Just updated this to match what's done in the worklist's `getEntityFilter()`: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/apps/patients/worklist/worklist_state.js#L134. 

Should pass every time now.